app/components/DocumentBreadcrumb.js

```diff
@@ -111,7 +102,7 @@ const DocumentBreadcrumb = ({ document, children, onlyText }: Props) => {
   if (onlyText === true) {
     return (
       <>
-        {collection.name}
+        {collection && collection.name}
```

Comment:
```suggestion
        {collection?.name}
```

I believe we can start to embrace the future